content/docs/start/experiments.md

```diff
@@ -21,39 +21,34 @@ the [`example-dvc-experiments`][ede] project.
 
 <details>
 
-### ⚙️ Installing the example project
+### ⚙️ Initializing a project with DVC experiments
```

Comment:
💅  Should be H2 I think?

Reply:
It was H3 before IIRC, but made it H2 now. Thanks.